packages/orbit-components/src/TextLink/helpers/twClasses.ts

```diff
@@ -23,3 +24,30 @@ export const typeClasses: Record<TYPE_OPTIONS, string> = {
   [TYPE_OPTIONS.WHITE]:
     "text-link-white-foreground hover:text-link-white-foreground-hover active:text-link-white-foreground-active",
 };
+
+export const alertDescendantClasses: Record<ALERT_TYPE, string[]> = {
```

Comment:
should these be in `TextLink`? 🤔 they're `Alert` specific, imo makes more sense to have em there. would just put them inside the main component for observability

Reply:
Good point. I thought thoroughly about this. In the end, I thought the opposite way: when we ever change TextLink styles, if we don't have these classes here… we _will_ forget about the impact such change can have in Alert. We won't remember looking for them inside Alert, will we?

Alert only changes _some_ of the TextLink styles, not all of them. So I am really not sure about the best approach here.

> changing Alert links will show up as changes in TextLink in Git history

I would worry about that if we had package per component versioning / changelog. Since it's not the case, I don't think that's problematic in the practical way (but I agree and understand your concern here).

Reply:
well the long term vision is to _not_ have this kind of cross-referencing in the future anyway, so let's not bother then. consider this resolved!